styles.css

```diff
@@ -91,3 +86,211 @@ ul {
   align-items: center;
   margin: auto;
 }
+
+/* css for the second part of the project */
+
+.works {
+  display: grid;
+  text-align: center;
+  justify-content: center;
+}
+
+.recent-works {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+}
+
+.img-wrapper {
+  height: 252px;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  background-color: #ebf0ee;
+}
+
+.card {
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  flex-direction: column;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  height: 474px;
+  width: 327px;
+}
+
+ul {
+  padding-left: 0;
+}
+
+.card-name {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  color: #3a4a42;
+  align-items: center;
+  justify-content: center;
+}
+
+.technologies {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  list-style-type: none;
+  gap: 15px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  color: #3a4a42;
+}
+
+.tech-tag {
+  background: #ebf0ee;
+  border-radius: 4px;
+  padding: 8px 12px;
+}
+
+.see-project {
+  padding: 12px;
+  gap: 10px;
+  width: 122px;
+  height: 48px;
+  background: #36b37f;
+  border-radius: 4px;
+  color: #fff;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  outline: 0;
+  border: 0;
+}
+
+.see-project:hover {
+  box-shadow: 8px 16px rgba(54, 179, 127, 0.24);
+}
+
+.see-project:active {
+  background: #008552;
+}
+
+.about-me {
+  align-items: center;
+  text-align: center;
+  background-image: url("about-me-background.png"), url("top-right-corner.png");
+  background-repeat: no-repeat, no-repeat;
+  background-position: left bottom, top right;
+}
+
+body .about-me-title {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+}
+
+.about-me-description {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  color: #42526e;
+}
```

Comment:
- [ ] Nice work on the "about me" section, the only missing detail is some margin around the paragraph 🔽 , it will improve the visibility of your paragraph and make it look good like in the Figma design.

![2023-03-25_173926](https://user-images.githubusercontent.com/104319462/227733122-e2983bc0-5ee7-4763-8515-e0668771cfa4.png)

Reply:
- [ ] As the previous reviewer suggested, Kindly work on it. :+1: 

> * [ ]  Nice work on the "about me" section, the only missing detail is some margin around the paragraph, it will improve the visibility of your paragraph and make it look good like in the Figma design.
> 
> ![2023-03-25_173926](https://user-images.githubusercontent.com/104319462/227733122-e2983bc0-5ee7-4763-8515-e0668771cfa4.png)